.circleci/config.yml

```diff
@@ -2,19 +2,20 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.12.3
+      - image: circleci/golang:1.13.6
```

Comment:
can we use 1.14?

Reply:
I've used 1.13 because I've tested vamp-policies against that version. I can bump up and check with 1.14 though.